Makefile

```diff
@@ -67,13 +69,16 @@ generate: controller-gen mockgen
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 	go generate ./...
 
-# Build the docker image
-docker-build: test
-	docker build . -t ${IMG}
-
-# Push the docker image
-docker-push:
-	docker push ${IMG}
+# covered by boilerplate XD
+# if they come back in the next version please check how to work around the issue
+#
+## Build the docker image
+#docker-build: test
```

Comment:
Boilerplate expects Dockerfile in `build/Dockerfile`. You can override this by setting `OPERATOR_DOCKERFILE`

Reply:
I tried to do my due diligance and try onboarding anyways

as seen here https://github.com/openshift/boilerplate/blob/4e0dabbee031072b635a169ee3e5dc12d0bc5168/boilerplate/openshift/golang-osd-operator/update#L16
it is hardcoded, so still needs some changes there aswell

Reply:
🤦